Add Submariner 2023 review (#1115)
cncf/toc: reviews/sandbox/2023/2023-submariner-annual.md

https://github.com/orgs/submariner-io/teams/submariner-owners

Committers:

* Aswin Suryanarayanan (@aswinsuryan, Red Hat)
* Daniel Farrell (@dfarrell07, Red Hat)
* Janki Chhatbar (@Jaanki, Red Hat)
* Mike Kolesnik (@mkolesnik, Red Hat)
* Vishal Thapar (@vthapar, Red Hat)

https://github.com/submariner-io/submariner/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/submariner-operator/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/admiral/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/cloud-prepare/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/lighthouse/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/subctl/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/shipyard/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/releases/blob/devel/CODEOWNERS.in
https://github.com/submariner-io/submariner-website/blob/devel/CODEOWNERS.in

Members:

* Andrew Stoycos (@astoycos, Red Hat)
* Brent Salisbury (@nerdalert, Red Hat)
* Maayan Friedman (@maayanf24, Red Hat)
* Nir Yechiel (@nyechiel, Red Hat)
* Noam Manos (@manosnoam, Red Hat)
* Masaki Kimura (@mkimuram, Hitachi)
* Maxim Babushkin (@MaxBab, Red Hat)

> * pkomarov (@pinikomarov, unknown)

Author: @skitt said this member is a Red Hatter and I confirmed. Will send an edit to fix.

> * pkomarov (@pinikomarov, Red Hat)

Reviewer: ```suggestion
* Pini Komarov (@pinikomarov, Red Hat)
```

Author: Yeah I thought about that but I didn't see his name and handle associated publically (although I didn't look closely) and didn't want to cause a privacy issue.

Author: I guess his handle is his name, if you know how to split it. I'll just add it.

Reviewer: Yeah that’s why I made the suggestion — the handle reveals all ;-).

* Steve Mattar (@SteveMattar, ex-Red Hat)
* Yossi Boaron (@yboaron, Red Hat)

https://github.com/orgs/submariner-io/people

## Adoption

> Submariner has adopters actively using it in production for a verity of usecases.

Reviewer 2: ```suggestion
Submariner has adopters actively using it in production for a variety of use cases.
```

We haven't discussed identifying users, but see the Add Public Adopters goal.

https://github.com/submariner-io/submariner/blob/devel/ADOPTERS.md

## Performance Against Past Goals

### Added Adopters

> One of Submariner's goals was to increase our number of adopters. We've accomplished this to a significant extent.

Reviewer 2: ```suggestion
One of our goals for Submariner was to increase the number of adopters. We've accomplished this to a significant extent.
```

We seem to have quite a few users doing a verity of interesting things with Submariner and subsets of Submariner.
We'll continue trying to grow Submariner's user community.

> ### Didn't Add Committers from not-Red Hat

Reviewer 2: ```suggestion
### Didn't Add Committers outside of Red Hat
```

Author: How about "from Groups other than"? I think that's a bit more clear and avoids the double negative.

Reviewer 2: sure

> One of Submariner's goals was to add Committers or Owners from groups other than Red Hat.  We have not accomplished this goal.

Reviewer 2: ```suggestion
Another goal for Submariner was to add Committers or Owners from groups other than Red Hat.  We have not accomplished this goal.
```

Author: I guess you're also trying to shift the ownership of the goal from SubM to "for SubM", which makes sense. I'll send something parallel to the first change.

> Another goals for Submariner was to add Committers or Owners from groups other than Red Hat. We have not accomplished this goal.

Reviewer 2: ```suggestion
Another goal for Submariner was to add Committers or Owners from groups other than Red Hat. We have not accomplished this goal.
```

Author: Good catch, thanks.
